accern_xyme/accern_xyme.py

```diff
@@ -741,6 +741,78 @@ def get_settings(self) -> Dict[str, Any]:
         assert self._settings is not None
         return self._settings
 
+    def get_timing(self, waiting: str) -> Dict[str, Any]:
+
+        nodes = self.get_nodes()
+        node_list = []
+        node_total = []
+        node_avg = []
+        node_name: str = []
+        dict_temp = {}
+
+        def del_waiting(node_time: list) -> List[Any]:
+            for pos, cur in enumerate(node_time):
+                for key, value in cur.copy().items():
+                    if value == waiting:
+                        del node_time[pos]
```

Comment:
don't del :)
return an `Iterable` and `yield` what you want to return and skip over what you don't want to return